File: 30 Days of Code/Day 14 - Scope/solution.py
	# Add your code here
    def computeDifference(self):
        maxDiff = 0
        
        for i in range(len(a)):     
            for j in range(i,len(a)):
                diff = abs(a[i] - a[j])
                if(maxDiff < diff):
                    maxDiff = diff
    
    
        self.maximumDifference = maxDiff

File: 30 Days of Code/Day 14 - Scope/solution.java
    
    public Difference(int[] elements){
        this.elements = elements;
    }

    public void computeDifference(){
        int maxDiff =0;
        int size = elements.length;
        
        for(int i = 0; i < size; i++){
            for(int j = i + 1; j < size; j++){
                int diff = Math.abs(elements[j] - elements[i]);
                if(maxDiff < diff){
                    maxDiff = diff;
                }
            }
        }
        this.maximumDifference = maxDiff;
        
        
    }
